exe/next.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 if [[ "${@}" == "--init" ]]; then
-  echo "doing a thing"
+  echo "Creating a symlink (Gemfile.next -> Gemfile)"
```

Comment:
@jnraine I think it's useful information for the end user. When I started using this gem I didn't realize that `Gemfile.next` was a symlink to `Gemfile`. The README doesn't mention that `Gemfile.next` is a symlink, so I thought it would be nice to share that info in the `--init` part.

Unless you want to move that info to the message below?

Reply:
Good point. Should we also output a message about injecting the `next?` method too?

Reply:
@jnraine I just updated the wording in the message. Maybe with the sentence about `Gemfile.next` it's clear... what do you think?